ngReact.js

```diff
@@ -9,17 +9,17 @@
 (function (root, factory) {
   if (typeof module !== 'undefined' && module.exports) {
     // CommonJS
-    module.exports = factory(require('react'), require('angular'));
+    module.exports = factory(require('react'), require('react-dom'), require('angular'));
   } else if (typeof define === 'function' && define.amd) {
     // AMD
-    define(['react', 'angular'], function (react, angular) {
-      return (root.ngReact = factory(react, angular));
+    define(['react', 'react', 'angular'], function (react, reactDOM, angular) {
```

Comment:
By this logic the above should be
```js
module.exports = factory(require('react'), require('react'), require('angular'));
```
instead of referencing `react-dom`. As it is now the actual renderers still live in the main `react` package. The `react-dom` project is basically empty except for `index` and `server`, both of which simply call back to the original package.

They way you have it set now for the AMD portion you're importing react twice. AMD users will be hit with all the deprecation warning around the stuff that will *eventually* be migrated to `react-dom`.